### PR TITLE
Make test green

### PR DESCRIPTION
**Pull request by clem21**

### CODE REVIEW

File: src/Migrations/Version20210422065835.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
```
Comment (lanfisis) [inline src/Migrations/Version20210422065835.php:1]: Is it normal to have a migration of Sylius core in this commit?

Comment (clem21) [reply src/Migrations/Version20210422065835.php:1]: It's true that it is weird but if I remove this migration then `symfony console doctrine:schema:validate` return an error

Comment (maximehuran) [reply src/Migrations/Version20210422065835.php:1]: I think you need to check the doctrine configuration with MariaDB. 
A pull request talk about this thing on Sylius : 
https://github.com/Sylius/Sylius/pull/12413

**Review by jacquesbh — APPROVED** [review]

**Comment by jacquesbh** [comment]
Thank you @clem21 ! 🎉

**Pull request merged**